.storybook/preview-head.html

```diff
@@ -11,3 +11,12 @@
     return str;
   }
 </script>
+<!-- workaround to get Jed into storybook -->
+
+<script>
+  const sprintfStr = (str, elem) => { return str } ;
+
+  window.Jed = {
+    sprintf:   sprintfStr
```

Comment:
Does it work well when running stroybook? shouldn't it be a function?

Reply:
Works well, it is sort of a workaround to get Jed work in storybook